docs/topics/gradle/get-started-with-jvm-gradle-project.md

```diff
@@ -28,7 +28,17 @@ To get started, first download and install the latest version of [IntelliJ IDEA]
 
 7. From the **Gradle DSL** list, select **Kotlin**.
 8. Select the **Add sample code** checkbox to create a file with a sample `"Hello World!"` application.
-9. Click **Create**.
+
+   > You can also enable the **Generate code with onboarding tips** option to add some additional useful comments to your
+   > sample code.
+   >
+   {type="tip"}
+
+9. You can specify a Gradle version for your project under the **Advanced Settings** section either by using Gradle Wrapper or a local installation of Gradle:
```

Comment:
Instead of having this as a step in the standard flow, can you put this content in a collapsible at the end of the section? Like this [example](https://www.jetbrains.com/help/kotlin-multiplatform-dev/multiplatform-create-first-app.html#explore-the-expect-actual-mechanism-optional).